WBooks/Books Table/BooksTableViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  BooksTableViewController.swift
+//  WBooks
+//
+//  Created by Carolina Arcos on 8/17/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class BooksTableViewController: UIViewController {
```

Comment:
The `BooksTableViewController` is used to show certain books, so it needs to get that information from somewhere.
Usually, the viewmodel has that information: which books should it show, and all the information of the book that needs to be shown (and only the info that needs to be shown).

So this controller should be created with a view model and the view model with the books information.
Instead of mocking here (like with the magic number 5 in the cells), you should create this as you will use it, and mock the parameters passed to this (that is to say, the values with which it is created).

In this way you can even try to see how it works with different books instead of being 5 which have the same information.